tests/service-catalog/test/servicecatalog_test.go

```diff
@@ -158,19 +162,26 @@ func TestHelmBrokerClusterAddonsConfiguration(t *testing.T) {
 	scClient, err := scc.NewForConfig(k8sConfig)
 	require.NoError(t, err)
 
-	// when
-	err = dynamicClient.Create(context.TODO(), addonsConfig)
-	require.NoError(t, err)
 	defer func() {
+		if t.Failed() {
+			namespaceReport := report.NewReport(t,
+				k8sConfig,
+				report.WithHB())
+			namespaceReport.PrintJsonReport("default")
```

Comment:
use empty string rather than "default", you can use `v1.NamespaceAll` const from k8s